pkg/oplabels/labels.go

```diff
@@ -17,16 +18,17 @@ const (
 	NameLabelValue             = "cassandra"
 	InstanceLabel              = "app.kubernetes.io/instance"
 	VersionLabel               = "app.kubernetes.io/version"
-	CreatedByLabel             = "app.kubernetes.io/created-by"
+	emptyString                = ""
+	labelCut                   = ".-_"
```

Comment:
In other parts of the code we replace "_" with "-" if possible. All of these characters are accepted values, they should be allowed (just not in the first character), in which case the regexp should probably just filter them and this field itself is not necessary.